app/hardware/server/assign/assign.component.js

```diff
@@ -62,5 +63,9 @@
     function assignClientModal() {
       return ServerAssignModal.client([assign.server], assign.server.access.sub);
     }
+
+    function hasSubSuperAccess() {
```

Comment:
since this is a function, you would need to use `assign.client.subSuperAccess()` instead of `assign.client.subSuperAccess` to get the returned value from the function.

We'll need this implemented for real, what's the plan here? We would need to fetch this off of the authenticated user's account details. Since we'd be using this in multiple places, we may need to add another service to get & cache this information, or maybe we can cache it with the user details (like user name)?